source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -57,17 +60,27 @@ class FilterConfig {
 
   const LocalInfo::LocalInfo& localInfo() const { return local_info_; }
 
+  Http::Code statusOnError() const { return status_on_error_; }
+
   Runtime::Loader& runtime() { return runtime_; }
 
   Stats::Scope& scope() { return scope_; }
 
   Http::Context& httpContext() { return http_context_; }
 
 private:
+  static Http::Code toErrorCode(uint64_t status) {
```

Comment:
What's the reason to constrain this to the 5xx range? Why not allow the user to pick whatever they want? Especially potentially another 4xx? Just curious. If this is what we want, please make the docs clear that the specified config is constrained and optimally use a validator.

Reply:
Since it is always related to an error, I thought that 5xx would be more correct, but I think we can make it more flexible.